pages/apim/3.x/event-native/apim-event-native-api-management-overview.adoc

```diff
@@ -0,0 +1,89 @@
+[[apim-event-native-api-management-overview]]
+= Gravitee Event-native API Management (BETA)
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_event_native_api_management_overview.html
+:page-folder: apim/event-native
+:page-layout: apim3x
+
+[label label-version]#New in version 3.19.0#
+
+== Event-native API Management via the new evented Gateway (BETA)
+
+Gravitee Platform release 3.19.0 has introduced the first version (BETA) of a new type of Gateway and API definition - an evented Gateway that enables event-native API management.
+
+The new event-native policy execution engine is based on a modern and fully reactive architecture designed to address a number of challenges users have been facing with the existing v3 engine, and to build the foundation of Gravitee's event-native platform by adding features such as native support for Pub/Sub (Publish-Subscribe) design and enabling policies at message level. The new API definition makes it possible for the Gateway to “know” what is necessary in order for it to handle asynchronous API use cases.
+
+The new policy execution engine mode provides the following capabilities:
+
+* The ability to execute policies in the exact order in which they have been placed in Policy Studio. This addreses some issues experienced by users related to the order in which policies are executed by the existing v3 engine where policies interacting with the Head part of the request are always executed first, even when placed in a different order in the Policy Studio during the design phase. With the new execution engine, it is now possible to apply logic on a head policy based on the payload of the request for example, to apply dynamic routing based on the request payload.
+* Proper isolation between platform-level policies and API-level policies execution. This ensures that platform-level policies are always executed before any API-level policies during the request stage, and after any API-level policies during the response stage.
+* Removal of the need to define a scope for policies (`onRequest`, `onRequestContent`, `onResponse`, `onResponseContent`).
+* The introduction of new API types to better differentiate REST APIs from Async APIs (Websocket, SSE, Webhook), as a foundation to fully support the execution of AsyncAPI.
+
+The BETA release does not provide UI support for these capabilities.
+
+NOTE: The event-native policy execution engine mode has been released as a BETA version - it is not recommended using it in production until it is released as a GA version at the end of 2022. The GA release will come with additional features - contact us if you want to know more about it and what new features we are bringing to API Management regarding Sync & Async APIs.
```

Comment:
I was thinking Jupiter (the new engine) was in beta in september and then GA in december, then SME alpha in september, beta in december and then GA in march.

Reply:
We don't have a proper definition of alpha / beta stage at the time so we should keep beta for now. SME would stay in Beta in december.